apps/dotcms-ui/src/app/view/components/_common/dot-add-to-menu/dot-add-to-menu.component.html

```diff
@@ -0,0 +1,63 @@
+<dot-dialog
```

Comment:
This is not a common reusable component... I think it should be somewhere else, maybe in the content types components?